packages/compass-collection/src/stores/collection-tab.ts

```diff
@@ -12,28 +12,17 @@ import type Collection from 'mongodb-collection-model';
 import toNs from 'mongodb-ns';
 import type { MongoDBInstance } from 'mongodb-instance-model';
 import type { ActivateHelpers } from 'hadron-app-registry';
+import { CollectionSubtabs, type CollectionSubtab } from '../types';
 
 export type CollectionTabOptions = {
   /**
    * Collection namespace
    */
   namespace: string;
   /**
-   * Initial query to be set in the query bar
+   * Collection sub tab
    */
-  initialQuery?: unknown;
-  /**
-   * Initial saved sggregation (stored on disk) to apply to the agg builder
-   */
-  initialAggregation?: unknown;
-  /**
-   * Initial aggregation pipeline to set in the agg builder
-   */
-  initialPipeline?: unknown[];
-  /**
-   * Initial stringified aggregation pipeline to set in the agg builder
-   */
-  initialPipelineText?: string;
```

Comment:
We should't remove those from tab options, these are different use-cases to a subtab